src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -77,20 +77,12 @@ public void setUp(@SysErr Capturable systemErr) {
     public void testJavadocTagsWithoutArgs() throws Exception {
```

Comment:
Why do we still have exceptions on tags if we updated all tags in `javadocTag` production rule?

Reply:
Yes, I think we need to remove this test completely, right?

Reply:
> Yes, I think we need to remove this test completely, right?

This doesn’t really answer the question. And no, we should not remove test, we should have no exceptions in test at all if we updated all affected rules with new tag grammar, unless there is some other issue with the grammar for these tags.

Reply:
Sorry, my fault, I missed understood your point. Yes, we still have some problems with grammar, but I just focused on descriptions, not the arguments itself, like missing the parameter in `@param` for example.

Reply:
So, to be clear, there is still issues with parsing some Javadoc tags, but it is not related to the description? If so, please see if we have an open issue for it, and if not, open an issue reporting the offending tags.

Reply:
These remaining tags have a problem with missing labels/arguments just like I mentioned before, but do we have a checker for this type of violations? If not, I think it's a new checker more than an issue.

Reply:
No, these are are parser errors.

Reply:
Done at https://github.com/checkstyle/checkstyle/issues/13043

Reply:
Thanks a lot, we can continue discussion there.